Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView+SwiftUI.swift

```diff
@@ -39,6 +39,7 @@ extension _ChatChannelAvatarView {
         }
 
         override public func setUpLayout() {
+            hostingController?.view.translatesAutoresizingMaskIntoConstraints = false
             embed(hostingController!.view)
```

Comment:
I searched our codabase for `embed(hostingController!.view)` and it found 3 cases. Why do we change it just for this one?

Reply:
Maybe the others are not tested yet :) I will check, thank you. :)

Reply:
Wow very thanks for this inside, I actually overlooked that `TAMIC = false` is being set in tests for SwiftUI views, which was part of strange fails when I was trying to figure out. `HostingVC.view` doesn't need to have this property set to false... I deleted the setting from tests and now we should be fine. :)